components/cam/src/physics/cam/physpkg.F90

```diff
@@ -969,8 +969,10 @@ subroutine phys_run1(phys_state, ztodt, phys_tend, pbuf2d,  cam_in, cam_out)
 #if (! defined SPMD)
     integer :: mpicom = 0
 #endif
-    integer(i8) :: beg_count                     ! start time for a chunk
-    integer(i8) :: end_count                     ! stop time for a chunk
+    integer(i8) :: beg_pcount                    ! start time for the process
+    integer(i8) :: end_pcount                    ! stop time for the process
+    integer(i8) :: beg_ccount                    ! start time for a chunk
+    integer(i8) :: end_ccount                    ! stop time for a chunk
```

Comment:
I think more verbose names would be good here, like "beg_chnk_cnt" and "beg_proc_cnt". When I was looking at the diffs below my eyes were initially missing where the "c" or "p" was added.

Reply:
Will make the change soon. Thanks for the suggestion.

Reply:
@whannah1 , done. Please take a look.